IFC4x3/Properties/p/PermissibleRoadLoad_0dZw4UYSDFVPrQibCUZruL/Documentation.md

```diff
@@ -0,0 +1 @@
+Permissible road traffic load / [t]
```

Comment:
```suggestion
Permissible road traffic load
```
units are infered or specified within the property

Reply:
Revised https://github.com/bSI-InfraRoom/IFC-Specification/pull/164/commits/ab85f083c2717d30ef42c6a47dcce352a5b16e89